common/traits/druid_forms.txt

```diff
@@ -73,9 +73,9 @@ master_cat_form = {
 	martial = 3
```

Comment:
Maybe buff combat rating to contend with pack form?

Reply:
no

Reply:
pussy